### ISSUE TITLE
auth function in ApiService (continued)

### ISSUE THREAD

**Issue by fippli** [skewlab/inferno-front#22]
Seem to me like /api/auth returns json:
![screen shot 2017-10-30 at 08 56 08](https://user-images.githubusercontent.com/2800451/32161332-71350eb8-bd55-11e7-9449-74e02866bb4e.png)

Maybe the function verifyRespone funciton always should consider the http status codes?

**Comment by jjojo** [comment]
Yep, on the backend it does. This is what shows if you go to the address on the backend. 
That could be a solution but it would men some kind of ifstatement on the verify response or something. But it should be possible to extract both status and JSON and pass along on the front end. Just didn’t know how

**Comment by fippli** [comment]
Sure, well look into it.
